bids/events/base.py

```diff
@@ -250,24 +250,25 @@ def __init__(self, base_dir, default_duration=0, default_amplitude=1,
         self.extra_columns = extra_columns
         self.sampling_rate = sampling_rate
 
-    def read(self, files=None, reset=True, **kwargs):
+    def read(self, file_directory=None, reset=True, **kwargs):
         ''' Read in and process event files.
         Args:
-            files (None, list): Optional list of event files to read from. If
-                None (default), will use all event files found within the
-                current BIDS project. Note that if `files` is passed, existing
-                event files within the BIDS project are completely ignored.
+            file_directory (None, str): Optional path to folder containing
```

Comment:
Oh, and re: patsy, this appears to be a Windows-specific issue. Not sure what the problem is, but the travis tests are all passing as of my latest commits, without the version locking in requirements.txt, so I think it's probably fine without that, unless you're having problems with it on linux or Mac.